settings.py

```diff
@@ -1476,6 +1476,11 @@
     "FRI_SMS_SHIFT": "custom.fri.api.shift_custom_content_handler",
     "FRI_SMS_OFF_DAY": "custom.fri.api.off_day_custom_content_handler",
     "UCLA_MESSAGE_BANK": "custom.ucla.api.ucla_message_bank_content",
```

Comment:
Your ticket didn't mention keeping this, but I assume we need to while they migrate their app?